### DISCUSSION TITLE
Where to change Twilio SID and AUTH KEY

### DISCUSSION THREAD

**Discussion by ThalesGnimavo** [chatwoot/chatwoot#5527] [Chatwoot APIs] [answered]
Hello,
Do you know where we can manually change twilio credentials sid and auth tokens ? It's not possible from the frontend and they have not been saved on .env file too. We search the path to the file on the server with no result.

Thank you
Juste

**Comment by sojan-official** [comment, marked as answer]
@justethales  ref: https://www.chatwoot.com/docs/product/channels/sms/twilio/ 
at the moment, we don't allow updating credentials post inbox creation. you will have to create a new inbox

**Comment by ThalesGnimavo** [comment]
Hello @sojan-official 
thank you. for response. Yes. I already switched to a new channel that I created but all the conversation history with 700+ users are on the old whatsapp channel. :-(
Anyway, we are using the powerful search box to find old message we need. 

thank you again
Juste
